crates/turborepo-lib/src/opts.rs

```diff
@@ -70,8 +70,8 @@ pub struct RunOpts<'a> {
     pub graph: Option<GraphOpts<'a>>,
     pub(crate) no_daemon: bool,
     pub(crate) single_package: bool,
-    pub log_prefix: LogPrefix,
-    pub log_order: LogOrder,
+    pub log_prefix: ResolvedLogPrefix,
+    pub log_order: ResolvedLogOrder,
```

Comment:
Post port we should change the CLI to take `Option<LogOrder>` and remove the `LogOrder::Auto` variant in favor of `None`. This removes the need for the duplicate `ResolvedLogOrder` definition.